src/aap_eda/settings/default.py

```diff
@@ -317,10 +317,6 @@ def _get_secret_key() -> str:
 
 RQ_STARTUP_JOBS = []
 RQ_PERIODIC_JOBS = [
-    {
```

Comment:
@benthomasson If this gets removed how would the periodic updates happen? The activations run as detached process and have no affinity to a worker they have affinity to the node where it is running. Any worker on that node can get the logs/status from an activation and take actions like stop/restart etc.

Reply:
There are actions that are not yet implemented in this PR.

Reply:
Monitor processes run instead of the RQ workers on the nodes.   A monitor process can monitor any number of activations.